File: SevaHands/SevaHands2/app/src/main/java/com/example/sevahands/services.java
package com.example.sevahands;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.google.android.material.bottomnavigation.BottomNavigationView;

public class services extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_services);

        BottomNavigationView bottomNavigationView = findViewById(R.id.nav_view);
        //bottomNavigationView.setSelectedItemId(R.id.navigation_Explore);

        bottomNavigationView.setOnItemSelectedListener(item -> {

            switch (item.getItemId()) {
               /* case R.id.navigation_Explore:
                    return true;
                    */


                case R.id.navigation_home:
                    startActivity(new Intent(getApplicationContext(), MainActivity.class));
                    overridePendingTransition(R.transition.right, R.transition.left);
                    finish();
                    return true;

               /* case R.id.navigation_contact:
                    startActivity(new Intent(getApplicationContext(), contact.class));
                    overridePendingTransition(R.transition.right, R.transition.left);
                    finish();
                    return true;

                */

                case R.id.navigation_donate:
                    startActivity(new Intent(getApplicationContext(),donate.class));
                    overridePendingTransition(R.transition.right, R.transition.left);
                    finish();
                    return true;





            }
            return false;
        });
    }
}

File: app/src/main/java/com/example/sevahandsversionone/Services.kt
package com.example.sevahandsversionone


import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.bottomnavigation.BottomNavigationView

class Services : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_services)

        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.nav_view)


        // Control the bottom navigation bar
        bottomNavigationView.setOnItemSelectedListener { item ->
            when (item.itemId) {
                R.id.navigation_home -> {
                    startActivity(Intent(applicationContext, Home::class.java))
                    overridePendingTransition(R.transition.right, R.transition.left)
                    finish()
                    true
                }
                R.id.navigation_contact -> {
                    // Start the Contact activity
                    startActivity(Intent(applicationContext, Contact::class.java))
                    overridePendingTransition(R.transition.right, R.transition.left)
                    finish()
                    true
                }
                R.id.navigation_dashboard -> {
                    // Start the Home activity
                    startActivity(Intent(applicationContext, DashBoard::class.java))
                    overridePendingTransition(R.transition.right, R.transition.left)
                    finish()
                    true
                }
                else -> false
            }
        }


        // Find the RecyclerView in your layout
        val recyclerView = findViewById<RecyclerView>(R.id.RC_View)

// Create an instance of the ServiceAdapter and pass your serviceItems
        val adapter = ServiceAdapter(serviceItems)

// Set the adapter for the RecyclerView
        recyclerView.adapter = adapter

// Set the layout manager for the RecyclerView (e.g., LinearLayoutManager)
        recyclerView.layoutManager = LinearLayoutManager(this)

    }
}
